src/react/features/lenses/timeline/hooks/getTimelineData.spec.js

```diff
@@ -0,0 +1,29 @@
+import { getTimelineData } from './getTimelineData'
+
+const zeroLocation = { lat: 0, lng: 0 }
+const dioryWithUndefinedDate = { date: undefined }
+
+describe('getTimelineData', () => {
```

Comment:
As a reader of this spec, I would like to read the spec 
- in plain english
- from top to bottom
- as little as possible

Therefore, I suggest that 
- we write the spec in plain English in "describe" and "it"
- we do not set variables in the beginning of the spec as one have to scroll to top to see what e.g. zeroLocations means
- e.g. diory: { date: undefined } already says to "diory with undefined date" -> shorter, no extra variables, less lines, more exact, no scrolling :)